governance/elevation.py

```
@@ -0,0 +1,257 @@
+               for r in load_ledger(ledger_dir)
+               if r.get("action") == "elevation.revoke"}
+    out = []
+    for r in load_ledger(ledger_dir):
+        if r.get("action") != "elevation.grant":
+            continue
+        p = payload_of(r)
+        eid = p.get("elevation_id")
+        if eid in revoked:
+            continue
+        exp = p.get("expires_at")
+        if exp and exp < iso(now):  # 字典序即时间序（同 ISO 格式）
```

Ruff (0.16.2)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)